src/sage/plot/contour_plot.py

```diff
@@ -1,3 +1,4 @@
+# sage.doctest: optional - sage.symbolic
```

Comment:
it's `optional` or `needs` ?

Reply:
Thanks, I've changed it to `# needs`. (The doctest fixer does not rewrite file-level tags.)